apps/vc-api/src/vc-api/exchanges/vp-submission-verifier.service.ts

```diff
@@ -113,10 +113,20 @@ export class VpSubmissionVerifierService implements SubmissionVerifier {
     const pex: PEX = new PEX();
 
     credentialQuery.forEach(({ presentationDefinition }, index) => {
-      const { errors: partialErrors } = pex.evaluatePresentation(
-        presentationDefinition,
-        presentation as IPresentation
-      );
+      let partialErrors;
```

Comment:
Maybe it's better to initialize the empty array and use `const`
```suggestion
      const partialErrors = [];
```

Reply:
AFAIR it forced next lines to be split into more steps as `{ errors: partialErrors } = pex.evaluatePresentation(` is not valid then. And it would require this that is less readable I think:
```typescript
const { errors } = pex.evaluatePresentation(
          presentationDefinition,
          presentation as IPresentation
        )
        
        partialErrors.push(...errors);
```